src/NBitcoin/ConsensusOptions.cs

```diff
@@ -112,6 +112,10 @@ public class PosConsensusOptions : ConsensusOptions
         /// <summary>Coinstake minimal confirmations softfork activation height for testnet.</summary>
         public const int CoinstakeMinConfirmationActivationHeightTestnet = 436000;
 
+        /// <summary>PoS block's timestamp mask.</summary>
+        /// <remarks>Used to decrease granularity of timestamp. Supposed to be 2^n-1.</remarks>
+        public const uint StakeTimestampMask = 0x0000000F;
```

Comment:
Could the name of this be `BlockTimestampMask` ? As it's part of `PosConsensusOptions` class so the caller will know the constant is part of Pos.

Reply:
Its a very specific POS related argument so fine like this.